client/lib/invites/actions.js

```diff
@@ -121,6 +121,10 @@ export function acceptInvite( invite, callback ) {
 			} else {
 				if ( ! get( invite, 'site.is_vip' ) ) {
 					dispatch( successNotice( ...acceptedNotice( invite ) ) );
+
+					if ( invite.role !== 'follower' && invite.role !== 'viewer' ) {
+						dispatch( receiveSites( data.sites ) );
```

Comment:
Is there a reason why the sites list shouldn't be updated when the site you're invited to is VIP? Did the original code behave the same way? I know absolutely nothing about VIP sites, so I can't tell.

Reply:
The only reason I put it there is that the success notice is also dispatched only if site is not VIP. Unfortunately, I don't know about VIP sites as well.

You are right that this is probably not what the original code did. It didn't distinguish between VIP and not. I'll update; good catch.